pypeit/core/flux.py

```diff
@@ -804,6 +802,38 @@ def load_extinction_data(longitude, latitude, toler=5. * units.deg):
     # Return
     return extinct[['wave', 'mag_ext']]
 
+def load_filter_file(filter):
+    """
+    Load a system response curve for a given filter
+
+    Args:
+        filter (str): Name of filter
+
+    Returns:
+        ndarray, ndarray: wavelength, instrument throughput
+
+    """
+    DES_filters = ['DES_{}'.format(i) for i in ['g','r','i','z','Y']]
```

Comment:
Thanks for developing this. We need to add more filters like SDSS, PS1, UKIRT/VISTA. We can leave it for now and add more options later.